Dockerfile

```diff
@@ -4,17 +4,22 @@ FROM hayd/deno:latest
 # Create and move into /bot directory
 WORKDIR /bot
 
-# Use user deno so the bot isn't running as root
-USER deno
+# Create a volume to store the database
+VOLUME /bot/db
+
+# Copy the source code
+COPY . .
 
-# Copy and cache all of the dependencies so they don't need to be downloaded every run
-COPY deps.ts .
-RUN deno cache deps.ts
+# Chown the directory and its content so that it belong to Deno user
+RUN chown -R deno:deno /bot
 
+# Use user deno so the bot isn't running as root
+USER deno
 
-# Copy all the rest of the files and type check them so they don't need to be checked every run
-ADD . .
-RUN deno cache mod.ts
+# Cache all of the dependencies so they don't need to be downloaded every run
+RUN cp /bot/configs.example.ts /bot/configs.ts && \
```

Comment:
The file "configs.example.ts" is copied as "configs.ts", this is intended?

Reply:
Yes. So that everything build and is "testable" without binding own config.
BTW you need to add a .dockerignore to ignore configs.ts so you don't copy it on the image once in a public Docker registry